x-pack/plugins/security_solution/public/resolver/models/resolver_tree.ts

```diff
@@ -42,7 +42,7 @@ export function lifecycleEvents(tree: ResolverTree) {
 /**
  * This returns a map of entity_ids to stats for the related events and alerts.
  */
-export function relatedEventsStats(tree: ResolverTree): Map<string, ResolverNodeStats> {
+export function relatedEventStats(tree: ResolverTree): Map<string, ResolverNodeStats> {
```

Comment:
I see your point. My motivation for changing this was just because I kept typing it as `relatedEventStats`. How do you feel about `nodeStats`? @bkimmel

Reply:
nvm, putting back the original name

Reply:
To be consistent with our naming in a few other places, should we add a `ByEntityId` at the end, too? Like `nodeStatsByEntityId`

Reply:
dunno